sdk/src/main/scala/com/horizen/api/http/BlockBaseApiRoute.scala

```diff
@@ -119,6 +119,24 @@ abstract class BlockBaseApiRoute[
     }
   }
 
+
+  /**
+   * Return here best sidechain block height in active chain
+   */
+  def getCurrentHeight: Route = (post & path("currentHeight")) {
```

Comment:
we need update `sidechainApi.yaml` file: add new method description

Reply:
done